<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuePanel</class>
 <widget class="QWidget" name="MainMenuePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">&lt;align='center'&gt;</string>
  </property>
  <widget class="QToolButton" name="button_vehicle_setup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Vechile setup wizzard</string>
   </property>
   <property name="statusTip">
    <string>Vechile setup wizzard</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../resources/setup.png</normaloff>../../../resources/setup.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="button_vehicle_configuration">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Vehicle configuration</string>
   </property>
   <property name="statusTip">
    <string>Vehicle configuration</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../resources/configuration.png</normaloff>../../../resources/configuration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="button_vehicle_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Vehicle status</string>
   </property>
   <property name="statusTip">
    <string>Vehicle status</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../resources/info.png</normaloff>../../../resources/info.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
